lib/logger/lib/logger.ex

```diff
@@ -19,6 +19,27 @@ defmodule Logger do
     * Wraps OTP's `error_logger` to prevent it from
       overflowing.
 
+  A common use case is tracking when an important function is called.
+
+      def solve(problem) do
+        Logger.info fn ->
+          "solve/1 called with problem #{inspect problem}"
+        end
+        solve(problem, [])
+      end
+
+  The `Logger.info/2` macro emits the provided message at the `:info`
+  level. There are additional macros for other levels. Refer to the `:level` and
```

Comment:
We can omit mentioning `:level` and `:compile_time_purge_level` IMO because they're really close in the next section, and doing so will bring down the noise a bit here.